<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>535</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Отрезки</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="mainview">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="cda">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ЦДА</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="br_float">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Брезенхем(float)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="br_int">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>70</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Брезенхем(int)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="br_smooth">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>100</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Брезенхем(сглаживание)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lib">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>130</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Библиотечный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bground">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать цвет фона</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="bground_color">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="line_color">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>210</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_line">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>200</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать цвет линии</string>
    </property>
   </widget>
   <widget class="QLabel" name="begin_point">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>250</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начальная точка</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_point">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>280</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Конечная точка</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="begin_x">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>250</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>711</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="begin_y">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>250</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>531</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="end_x">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>280</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>711</number>
    </property>
    <property name="value">
     <number>355</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="end_y">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>280</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>531</number>
    </property>
    <property name="value">
     <number>265</number>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_line">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отобразить линию </string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_sun">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отобразить пучок </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_angle">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>390</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="diametr">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Диаметр пучка</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle_step">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>390</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Угловой шаг</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>450</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_dia">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>360</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_all">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>460</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить холст</string>
    </property>
   </widget>
   <widget class="QLabel" name="diametr_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время выполнения:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
